packages/govuk-frontend-review/src/views/examples/js-support-error/index.njk

```diff
@@ -0,0 +1,34 @@
+{% extends "layouts/layout.njk" %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/accordion/macro.njk" import govukAccordion %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    href: "/"
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">Example: JavaScript error - Support Error</h1>
+  <p class="govuk-body-lead">
+      Open your <a href="https://developer.mozilla.org/en-US/docs/Web/API/console#see_also">brower's console</a>
+      to witness the error thrown in browsers where GOV.UK Frontend is not supported.
+    </p>
+  {{ govukAccordion()}}
+{% endblock %}
+
+{% block bodyEnd %}
+  <!-- Simulate GOV.UK Frontend not being supported -->
+  <script>
+    document.body.classList.remove('govuk-frontend-supported')
+  </script>
+  <script type="module" src="/javascripts/govuk-frontend.min.js"></script>
+  <!-- We don't want to use `initAll` as it returns early if govuk-frontend is not supported -->
+  <script type="module">
+    import { Accordion } from '/javascripts/govuk-frontend.min.js'
+
+    const $element = document.querySelector('[data-module="govuk-accordion"]');
```

Comment:
Probably don't need `$element` as it's `null` anyway

The support checks run before the "instanceof" check

Reply:
That's technically what happens, but I'd rather keep the initialisation code as close as possible to what may happen in actual code for clarity.

Reply:
I think I'd wonder where the Accordion was?

Reply:
We're still importing `Accordion` and running `new Accordion($element)`?

I'd prefer we used `$element = null` until we expand this page with a component

See https://github.com/alphagov/govuk-frontend/pull/4075#discussion_r1308667562

Reply:
Thanks for fixing this